spec/mixlib/install/backend/package_router_spec.rb

```diff
@@ -125,6 +125,23 @@
     end
   end
 
+  context "for a 32 bit version that is mapped to i686" do
+    let(:channel) { :stable }
+    let(:product_name) { "chef" }
+    let(:product_version) { "12.4.1" }
+    let(:platform) { "el" }
+    let(:platform_version) { "5" }
+    let(:architecture) { "i386" }
```

Comment:
This doesn't test any remapping as is right now. An `omnibus.architecture` of `i686` should be re-mapped to `i386`. This test just says `i386` architecture remaps to `i386`.

Take a look at some example properties from a 32-bit EL6 build.
http://artifactory.chef.co/webapp/#/artifacts/browse/tree/Properties/omnibus-stable-local/com/getchef/chef/12.14.89/el/6/chef-12.14.89-1.el6.i386.rpm